tests/migration/test_migration.py

```diff
@@ -1,114 +1,76 @@
 import os
 import subprocess
-from contextlib import closing
 
-import pkg_resources
-import psycopg2
 import pytest
-from psycopg2 import sql
-from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
 
-from procrastinate import __version__, aiopg_connector, schema
-
-PG_SERVICES = """
-[procrastinate_test1]
-dbname=procrastinate_test1
-[procrastinate_test2]
-dbname=procrastinate_test2
-"""
-
-
-def _execute_sql(cursor, query, *identifiers):
-    cursor.execute(
-        sql.SQL(query).format(
-            *(sql.Identifier(identifier) for identifier in identifiers)
-        )
-    )
+from procrastinate import aiopg_connector, schema
 
+BASELINE = "0.5.0"
 
+# Pum config file is currently broken (https://github.com/opengisch/pum/issues/5)
+# When it's fixed, we can add a config file here, and then get rid of omit_table_dir
 @pytest.fixture
-def migrations_directory():
-    return pkg_resources.resource_filename("procrastinate", "sql/migrations")
+def pum():
 
+    env = {**os.environ, "PGSERVICEFILE": "tests/migration/pgservice.ini"}
```

Comment:
@elemoine This way, we don't even need to name the fixture

Reply:
in case you doubt,
```pycon
Python 3.6.9 (default, Dec 14 2019, 18:23:02)
[GCC 4.2.1 Compatible Apple LLVM 11.0.0 (clang-1100.0.33.8)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> {**{1: 2}, 3:4}
{1: 2, 3: 4}
>>> {**{3: 2}, 3:4}
{3: 4}
```

Reply:
I keep forgetting about this syntax. Looks good.